src/cmd/flux-kvs.c

```diff
@@ -116,6 +143,13 @@ static struct optparse_subcommand subcommands[] = {
       0,
       dir_opts
     },
+    { "ls",
+      "[-R] [-d] [-w COLS] [-1] [-W] [-C] [key...]",
+      "List directory",
```

Comment:
also noticed the manpage is [-R] [-d] [-C] [-1|-C] [-w COLS] [-F], should this match up together (most notably [-1|-C].

Reply:
Indeed.